myair_client/const.py

```diff
@@ -0,0 +1,37 @@
+VERSION = "0.0.0-dev"  # Version updated by release workflow
+
+DOMAIN = "resmed_myair"
+DEFAULT_UPDATE_RATE_MIN = 30
+
+AUTHN_SUCCESS = "SUCCESS"
+AUTH_NEEDS_MFA = "MFA_REQUIRED"
+
+# Config keys
+CONF_USER_NAME = "Username"
+CONF_PASSWORD = "Password"
+CONF_REGION = "Region"
+CONF_DEVICE_TOKEN = "device_token"
+CONF_VERIFICATION_CODE = "verification_code"
+
+REGION_NA = "NA"
+REGION_EU = "EU"
+
+KEYS_TO_REDACT: list[str] = [
```

Comment:
Sorry I should have been more explicit: the entire contents of the myair_client folder are a copy from https://github.com/prestomation/resmed_myair_sensors

I'd rather not make edits there since it's likely I'll have to refresh the folder from time to time.